src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ResponseDrain.cs

```diff
@@ -162,7 +160,15 @@ private static string GetResponseForContentMode(string content, ContentMode mode
                     string response = GetResponseForContentMode(content, mode);
                     await server.AcceptConnectionAsync(async connection =>
                     {
-                        await connection.ReadRequestHeaderAndSendCustomResponseAsync(response);
+                        // Process the first request, with some introduced delays in the response to
+                        // stress the draining.
+                        await connection.ReadRequestHeaderAsync().ConfigureAwait(false);
+                        foreach (char c in response)
```

Comment:
Could use DribbleStream here.  Maybe not worth it.

Reply:
I started down that path, but it didn't have existing Read/ReadAsync/CopyToAsync implementations, and decided this was just easier.

Reply:
Ok